docs/development/affiliated-packages.rst

```diff
@@ -231,6 +231,50 @@ You can find out what the latest version of astropy-helpers is by checking the
 `astropy-helpers <https://pypi.python.org/pypi/astropy-helpers/>`__ entry on
 PyPI.
 
+Customizing the documentation CSS
+---------------------------------
+
+As described in the documentation configuration file (`teplate/docs/conf.py
+<https://github.com/astropy/package-template/blob/master/docs/conf.py#L95>`_),
+the documentation uses a custom theme based on `bootstrap
+<http://getbootstrap.com/css/>`_. You can swap out this theme by editing the
+configuration file. You can also tweak aspects of the documentation theme by
+creating a custom CSS file in your package documentation.
+
+To do this, create a new CSS file in ``<packagename>/_static/`` -- let's call it
+``custom.css``::
```

Comment:
What about calling it ``<packagename>.css``? Some of the affiliates doing that already, e.g. photutils, ccdproc.